compiler/src/main/kotlin/walkers/grapher.kt

```diff
@@ -0,0 +1,359 @@
+package dk.aau.cs.d409f19.cellumata.walkers
+
+import dk.aau.cs.d409f19.cellumata.ast.*
+import java.io.OutputStream
+import java.io.PrintStream
+
+class ASTGrapher(sink: OutputStream, private val output: PrintStream = PrintStream(sink)) : BaseASTVisitor() {
+    var depth = 0
+
+    private fun printNode(from: Any, to: Any) {
+        output.println("\t" + from.hashCode() + " -> " + to.hashCode())
+    }
+
+    private fun printLabel(node: Any, label: String) {
+        output.println("\t" + node.hashCode() + " [label=\"" + label + "\"]")
+    }
+
+    override fun visit(node: WorldNode) {
+        var builder = StringBuilder()
+            .append("WorldNode\\nsize = ")
+            .append(node.dimensions.map {
+                when(it.type) {
+                    WorldType.EDGE -> it.size.toString() + "[edge=" + "]"
```

Comment:
I think you are missing a the state name in `"[edge=" + "]"`

Reply:
fixed in 398bea9